setup.py

```diff
@@ -62,6 +63,9 @@ def run(self):
               '-b {}'.format(current_version)])
         VERSION = get_version()
 
+with open(COMMIT_FILE, 'w') as cf:
+    cf.write(get_current_commit() + '\n')
+
```

Comment:
Travis python 2.7 complains is we use `print(get_current_commit, file=cf)` as print is not a function